packages/composables/src/cms/useCmsElementImage.ts

```diff
@@ -18,7 +18,7 @@ export type UseCmsElementImage = {
   containerStyle: ComputedRef<CSSProperties>;
   anchorAttrs: ComputedRef<AnchorHTMLAttributes>;
   imageAttrs: ComputedRef<ImgHTMLAttributes>;
-  imageContainerAttrs: ComputedRef<CSSProperties | ImgHTMLAttributes>;
+  imageContainerAttrs: ComputedRef<{ [k: string]: string }>;
```

Comment:
I do not like this change - it's more implicit, and we should aim for explicitly - there should not be just "string: string" - we should know what we can expect from this field

Reply:
@mdanilowicz I can agree with that :top: